### ISSUE TITLE
Xcode project Wont Build: _pb_release is reference but not included

### ISSUE THREAD

**Issue by jehartley** [firebase/firebase-ios-sdk#6569]
This is what Xcode is showing after failed build:
Undefined symbols for architecture x86_64:
  "_pb_release", referenced from:
      -[FIRCoreDiagnosticsLog dealloc] in FirebaseCoreDiagnostics(FIRCoreDiagnostics.o)
      -[GDTCCTUploader handleURLResponse:data:error:target:storage:batchID:] in GoogleDataTransport(GDTCCTUploader.o)
      -[GDTCCTUploader constructRequestProtoWithEvents:] in GoogleDataTransport(GDTCCTUploader.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

I am using the latest Cocoapods

**Comment by maksymmalyhin** [comment]
@jehartley Sorry you have an issue. Could you please populate the bug template to provide information required for further investigation.

**Comment by maksymmalyhin** [comment]
As the first steps I would recommend re-installing Cocoapods, cleaning the project and removing derived data.

**Comment by jehartley** [comment]
I have done all of that...re-installed Cocoapods, cleaning project, derived data delete, etc. Nothing works.

**Comment by jehartley** [comment]
How do I populate the bug template?

**Comment by morganchen12** [comment]
Fill out the information in [this file](https://raw.githubusercontent.com/firebase/firebase-ios-sdk/master/.github/ISSUE_TEMPLATE/bug_report.md). Make sure you provide a sample project that reproduces the linker issue you're seeing.

**Comment by wilhuff** [comment]
`pb_release` is only defined when nanopb is built with `PB_ENABLE_MALLOC`. We enable that [in the podspec](https://github.com/google/nanopb-podspec/blob/master/nanopb.podspec#L19).

Somehow your project has gotten into a state where this flag is missing on the nanopb pod. Either manually check the nanopb target in the pods project or `pod deintegrate`, remove Pods and pod-related projects, and then reinstall.

**Comment by jehartley** [comment]
> `pb_release` is only defined when nanopb is built with `PB_ENABLE_MALLOC`. We enable that [in the podspec](https://github.com/google/nanopb-podspec/blob/master/nanopb.podspec#L19).
> 
> Somehow your project has gotten into a state where this flag is missing on the nanopb pod. Either manually check the nanopb target in the pods project or `pod deintegrate`, remove Pods and pod-related projects, and then reinstall.

I tried this already. Deleted everything. reinstalled. Same thing.

**Comment by wilhuff** [comment]
In that case, we're really going to need a sample project that demonstrates the error. This is a assuredly a local project configuration problem.

**Comment by jehartley** [comment]
Well, I don't know what is wrong with my project. I made a brand new project. Installed Cocoapods, installed the two pods I installed in my other project, and it worked fine. I don't know what the difference is, but I guess I will just install them manually in my project. That worked before. Thanks for the help, everyone.

**Comment by jehartley** [comment]
Okay, I figured it out. My project ran its frameworks manually for years. So, I wanted to start using Cocoapods instead. I thought I deleted all of the manual frameworks, but I missed one...you guessed it, the nanopb. Once I deleted that, pod disintegrate, deleted everything, then reinstalled Cocoapods, reinstalled the framework pods, etc, everything worked fine.

I wouldn't have thought to do that, if not for wilhuff, so thank you, sir. And thanks to everyone for your input/help!!!
